src/python/pants/backend/python/typecheck/pytype/subsystem.py

```diff
@@ -21,7 +21,7 @@ class Pytype(PythonToolBase):
     )
 
     register_interpreter_constraints = True
-    default_interpreter_constraints = ["CPython>=3.7,<3.10"]
+    default_interpreter_constraints = ["CPython>=3.7,<3.11"]
```

Comment:
@idan-at do you have insight into whether this is an okay adjustment? It aligns these interpreter constraints (used for actually executing pytype?) with the ones that were used to generate the lockfile.

Reply:
I remember I checked that specific pytype version manifest for the supported Python versions, but If the lockfile resolves it differently, I might have mistaken

Reply:
Ah, okay, it looks like the lockfile version matches most closely: https://github.com/google/pytype/blob/33c477af9e5ca51933ef9780e8fd91f606577a64/setup.cfg#L30

So, I think this is good to go.